plone-5/instance/src/ukstats.sparql_dataconnector/src/ukstats/sparql_dataconnector/adapter.py

```diff
@@ -24,10 +24,12 @@ class SPARQLDataProviderForConnectors(object):
     def __init__(self, context, request):
         self.context = context
         self.request = request
+        
 
     @timing
     def _get_data(self):
         endpoint_url = self.context.endpoint_url
+        title = self.context.title
```

Comment:
best not to declare this outside the scope of the error catch, otherwise it's declared every time and only used once in a blue moon.